src/scripts/add-post-to-announcements.ts

```diff
@@ -0,0 +1,38 @@
+import yargs from 'yargs'
```

Comment:
We should probably remove this script before merging.

Reply:
couldn't it be useful (maybe implemented in a proper way) to post stuff to the announcements actor manually?

even then, it can be useful for debugging in staging. is staging strictly the `main` branch or can it include these sort of changes?

Reply:
I'm not sure actually. Having it be an admin-only HTTP API (POST `/v1/@announcements@social.distributed.press/outbox`) could be useful for that. It's just that scripts like this can only run when the server is shut down so we should be very sparing with how much functionality requires that. So far I think they've just been there for initial server setup.

Reply:
well, i didn't implement the POST /outbox as we decided that was out of scope lol

so uhh, your call, do i make the CLI changes to keep the script around or not?

Reply:
Lets use it for testing for now then remove before release and reivist again with the post to outbox use case.